examples/solids/elasticity.c

```diff
@@ -0,0 +1,713 @@
+// Copyright (c) 2017, Lawrence Livermore National Security, LLC. Produced at
+// the Lawrence Livermore National Laboratory. LLNL-CODE-734707. All Rights
+// reserved. See files LICENSE and NOTICE for details.
+//
+// This file is part of CEED, a collection of benchmarks, miniapps, software
+// libraries and APIs for efficient high-order finite element and spectral
+// element discretizations for exascale applications. For more information and
+// source code availability see http://github.com/ceed.
+//
+// The CEED research is supported by the Exascale Computing Project 17-SC-20-SC,
+// a collaborative effort of two U.S. Department of Energy organizations (Office
+// of Science and the National Nuclear Security Administration) responsible for
+// the planning and preparation of a capable exascale ecosystem, including
+// software, applications, hardware, advanced system engineering and early
+// testbed platforms, in support of the nation's exascale computing imperative.
+
+//                        libCEED + PETSc Example: Elasticity
+//
+// This example demonstrates a simple usage of libCEED with PETSc to solve
+//   elasticity problems.
+//
+// The code uses higher level communication protocols in DMPlex.
+//
+// Build with:
+//
+//     make elasticity [PETSC_DIR=</path/to/petsc>] [CEED_DIR=</path/to/libceed>]
+//
+// Sample runs:
+//
+//     ./elasticity -problem linElas -degree 2 -nu 0.3 -E 1 -forcing mms -mesh ./meshes/cylinder8_672e_4ss_us.exo
+//     ./elasticity -problem hyperSS -ceed /cpu/self -degree 2 -nu 0.3 -E 1 -forcing mms -mesh ./meshes/cylinder8_672e_4ss_us.exo
+//     ./elasticity -problem hyperFS -ceed /gpu/occa -degree 2 -nu 0.3 -E 1 -forcing mms -mesh ./meshes/cylinder8_672e_4ss_us.exo
```

Comment:
Do these work for you? I'm getting `DIVERGED_FNORM_NAN`.

Reply:
That should be `-problem linElas`. My mistake.